src/component/panels/PeaksPanel/PeaksPanel.tsx

```diff
@@ -207,6 +208,21 @@ function PeaksPanelInner({
           >
             <SvgNmrPeaks style={{ pointerEvents: 'none', fontSize: '12px' }} />
           </ActiveButton>
+
+          <ActiveButton
+            style={{ marginLeft: '2px', marginRight: '2px' }}
+            popupTitle={
+              peaksViewState.displayingMode === 'group'
+                ? 'Single Mode'
+                : 'Group mode'
```

Comment:
From a user's perspective, I'm not sure these two names make much sense. In the new mode, the peaks don't look like they are grouped. It's more like "spread".

Reply:
actually i did not find a good name this is why i use `group`, any suggestion it will be helpful